examples/multiphysics/sloshing-in-rectangular-tank/sloshing_0002/sloshing-in-rectangular-tank_Re0002.prm

```diff
@@ -24,19 +26,11 @@ subsection FEM
 end
 
 #---------------------------------------------------
-# Restart
+# Timer 
```

Comment:
Do we need the timer?

Reply:
I like the timer, but that's just me :).